docker/openstack-client-centos/run.sh

```diff
@@ -117,8 +120,12 @@ if [ -f ~/.ansible.cfg ]; then
   ANSIBLE_CFG="-v $HOME/.ansible.cfg:/root/.ansible.cfg"
 fi
 
+if [ "${COMMAND}" != "" ]; then
+  docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} --entrypoint="${COMMAND}" ${OPENSTACK_CLIENT_IMAGE}
+else
 
-echo "Starting OpenStack Client Container...."
-echo
-#docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack:z ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} ${OPENSTACK_CLIENT_IMAGE}
-docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} ${OPENSTACK_CLIENT_IMAGE}
+  echo "Starting OpenStack Client Container...."
+  echo
+  #docker run -it ${HOST_NET} ${REMOVE_CONTAINER_ON_EXIT} -v ${OPENSTACK_CONFIG_DIR}:/root/.config/openstack:z ${REPOSITORY_VOLUME} ${SSH_VOLUME} ${ANSIBLE_CFG} ${OPENSTACK_CLIENT_IMAGE}
```

Comment:
While you are at it, can we remove the commented out command that's no longer used?